src/search/FindInFiles.js

```diff
@@ -417,15 +418,26 @@ define(function (require, exports, module) {
         }
         
         var scopeName = searchModel.scope ? searchModel.scope.fullPath : ProjectManager.getProjectRoot().fullPath,
-            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query);
+            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query),
+            searchDeferred = new $.Deferred();
         
-        return candidateFilesPromise
+        candidateFilesPromise
             .then(function (fileListResult) {
                 // Filter out files/folders that match user's current exclusion filter
                 fileListResult = FileFilters.filterFileList(filter, fileListResult);
+                fileListResult = FindUtils.getSortedFiles(fileListResult, function (file) { return file.fullPath; }, FindUtils.getOpenFilePath());
```

Comment:
That is great, and it makes more sense. I am guessing that if 1.8s is still too much, then we would need to keep a sorted copy of the file list. Maybe instead of sorting the entire list, we could copy the list, grab the first 50 files, remove them from the list and repeat this on each process, which might be a bit faster than sorting the entire list?